README.md

```diff
@@ -1,4 +1,113 @@
 # Cognite `extractor-utils` REST extension
```

Comment:
is this published in pyPI, if so can you add an installation step here?

Reply:
Not yet :) Will add it when we do!